lib/views/shared/receive.dart

```diff
@@ -134,12 +134,27 @@ class ReceiveSharedState extends ChangeNotifier {
         switch (error.errorCode) {
           case ErrCodeTransferRejected:
             this.currentState = ReceiveScreenStates.TransferRejected;
+            this.error = "Description description ErrCodeTransferRejected";
```

Comment:
@shareef-dweikat can we push all text to main/lib/constants/app_constants.dart file to have one place for edit?

Reply:
@donpui Sorry i pushed AndroidManifest.xml by accident MainActivity.kt

Reply:
Yeah sure. This branch is only for app testings. If everything is working well, will invest more time to clean the code.